### PR TITLE
Fixed RTL navigations

### PR DESCRIPTION
**Pull request by sag1v**
This is due to #774

### CODE REVIEW

**Comment by majapw** [comment]
Hi @sag1v, I think we'll have to update the phrases that get read out by the screen reader as well. Do the keyboard shortcuts still make sense with this update?

**Comment by sag1v** [comment]
Hi @majapw, what do you mean by keyboard shortcuts? Iv'e only updated the logic inside the switch statement of the key press for right and left arrows, if we on RTL then the next focused day is calculated in reversed

**Comment by sag1v** [comment]
@majapw Any further thoughts about this PR?

File: src/components/DayPicker.jsx [hunk 1]
```diff
@@ -265,7 +265,7 @@ class DayPicker extends React.Component {
         break;
       case 'ArrowLeft':
         e.preventDefault();
-        newFocusedDate.subtract(1, 'day');
+        !isRTL ? newFocusedDate.subtract(1, 'day') : newFocusedDate.add(1, 'day');
```
Comment (majapw) [inline src/components/DayPicker.jsx]: I think our linter rules require this to be:
```
if (isRTL) {
  newFocusedDate.add(1, 'day');
} else {
  newFocusedDate.subtract(1, 'day');
}
```

File: src/components/DayPicker.jsx [hunk 2]
```diff
@@ -286,7 +286,7 @@ class DayPicker extends React.Component {
         break;
       case 'ArrowRight':
         e.preventDefault();
-        newFocusedDate.add(1, 'day');
+        !isRTL ? newFocusedDate.add(1, 'day') : newFocusedDate.subtract(1, 'day');
```
Comment (majapw) [inline src/components/DayPicker.jsx]:
```
if (isRTL) {
  newFocusedDate.subtract(1, 'day');
} else {
  newFocusedDate.add(1, 'day');
}
```

**Comment by ljharb** [comment]
Also, please rebase on the command line so as to remove any merge commits :-)

**Comment by sag1v** [comment]
@majapw 

> Can you also update the Home and End shortcuts?  
What kind of update? `Home` & `End`  are not affected by `RTL` as far as i know

**Comment by ljharb** [comment]
Ah, you’d have to clone your *fork* and rebase your branch there. No worries tho, we can do it for you prior to merging.

**Comment by majapw** [comment]
@sag1v While on keyboards that have a home and end button, i agree that it's not as much of an issue, but on keyboards where the functionality is forced by `fn + left` or `fn + right`, it feels like it might make sense to switch them? 

It might be a minority of keyboards for those accessing the RTL version of the datepicker, so maybe it doesn't make sense and home pointing to beginning of the week/end pointing to the end of the week is fine.

**Comment by sag1v** [comment]
@majapw Thanks, i see your point. If there is anyway to determine if a Home or End buttons aren't available then we can write the condition for that. Otherwise i think we should merge this PR to at least support the Arrow keys on `RTL` until  we can find a way to support fn + Arrows.

**Comment by ljharb** [comment]
I believe on a Mac, `fn` + left/right is indistinguishable from "home"/"end"?

**Comment by sag1v** [comment]
@ljharb 

> I believe on a Mac, fn + left/right is indistinguishable from "home"/"end"?


It seems [it's not that different than other brands](https://www.macworld.co.uk/how-to/mac/find-home-end-buttons-on-mac-3618360/):

**Pull request merged**